lua/cmp/view/ghost_text_view.lua

```diff
@@ -26,9 +26,13 @@ ghost_text_view.new = function()
   self.entry = nil
   vim.api.nvim_set_decoration_provider(ghost_text_view.ns, {
     on_win = function(_, win)
-      return win == self.win
-    end,
-    on_line = function(_, _, _, on_row)
+      if self.extmark_id then
+        vim.api.nvim_buf_del_extmark(self.extmark_buf, ghost_text_view.ns, self.extmark_id)
+        self.extmark_id = nil
+      end
+      if win ~= self.win then
+        return
+      end
```

Comment:
This temporary because inline ghost text was broken in cmp when this was forked.